zirgen/dsl/passes/GenerateAccum.cpp

```diff
@@ -321,7 +321,7 @@ struct GenerateAccumPass : public GenerateAccumBase<GenerateAccumPass> {
   void runOnOperation() override {
     getOperation().walk([&](ComponentOp component) {
       llvm::StringRef baseName = component.getName();
-      if (baseName.ends_with("$accum") || (baseName != "Top" && !baseName.starts_with("test$")))
+      if (baseName.ends_with("$accum") || !Zhlt::isEntryPoint(component))
```

Comment:
:+1:

Reply:
Is the check for `baseName.ends_with("$accum")` redundant here? Looks like the `isEntryPoint` function already checks for that case.

Reply:
If the name ends with `$accum` then it is an entry point. that check is included so that we don't get stuck in a loop generating accum code from accum code, which was a thing I saw happen during development 😬 

This is a good hint that I should add a comment here though. Thanks!